README.md

```diff
@@ -78,3 +80,9 @@ The logs are stored in `logs` subdirectory of the location where your `spatialos
 [Server worker configuration](docs/reference/server-worker-configuration.md)
 
 [Client worker configuration](docs/reference/client-worker-configuration.md)
+
+## Migration Guide
+[FAQ](docs/migration-guide/faq.md)
```

Comment:
Are we planning to add a migration guide to replace this at some point?

Reply:
The plan is to have a `migration guide` section containing multiple articles.